terraform/Makefile

```diff
@@ -92,7 +92,7 @@ apply: prep ## Have terraform do the things. This will cost money.
 		-input=false \
 		-refresh=true \
 		-var-file="$(VARS)"
-	echo "If that succeeded, run 'terraform output --json | (cd ../deploy-tool/ ; go run main.go )'"
+	echo "If that succeeded, navigate to deploy-operator, run 'make deploy; make install'. Then use 'kubectl apply -f crds.yml' on the file generated by terraform."
```

Comment:
I'm concerned about the proliferation of tools we have to remember to run *after* `make apply`. jsha has been looking into using a Helm chart to deploy Prometheus; could we do something similar for the deploy-operator and the CRDs? If not, let's add these `make` and `kubectl` steps to the `make apply` target.